README.md

```diff
@@ -28,7 +28,7 @@ go build -o rpc ./cmd/main.go
 
 #### As Library: 
 ```
-go build -buildmode=c-shared -o rpc.so ./cmd   
+go build -buildmode=c-shared -o librpc.so.1.0 ./cmd   
```

Comment:
Why are we renaming this?

Reply:
to follow linux standard .so library format. is there some existing Linux usage that is not documented?

Reply:
I ended up taking off the version ID part so the example is less brittle. On a standard linux distribution and install, the .so get a lot of links created in the common library file with version extenstions. That's way more of an end user situation than the sample needs actually, but for me coming from Linux, I was expecting to see a librpc.so file somewhere in the project. Thanks.